### PR TITLE
Completed default email template FR and EN translations

### PR DESCRIPTION
**Pull request by roadster31**

### CODE REVIEW

File: setup/insert.sql [hunk 1]
```diff
@@ -7765,22 +7765,22 @@ INSERT INTO `message_i18n` (`id`, `locale`, `title`, `subject`, `text_message`,
     (4, 'en_US', 'Mail sent to the customer when its password or email is changed in the back-office', 'Your account information on {config key=\"store_name\"} has been changed.', NULL, NULL),
     (5, 'en_US', 'Mail sent to the customer when its account is created by an administrator in the back-office', 'A {config key=\"store_name\"} account has been created for you', NULL, NULL),
     (6, 'en_US', 'Mail sent to an administrator who requested a new password', 'New password request on %store', NULL, NULL),
-    (7, 'en_US', 'Newsletter subscription confirmation mail', 'Your subscription to %store newsletter', NULL, NULL),
-    (8, 'en_US', 'Mail sent to the customer to confirm its account', 'Confirm your %store account', NULL, NULL),
+    (7, 'en_US', 'Newsletter subscription confirmation mail', NULL, NULL, NULL),
```
Comment (bibich) [inline setup/insert.sql]: There's a problem here. try to regenerate the `sql` file please.

File: setup/update/sql/2.3.0-alpha2.sql [hunk 2]
```diff
@@ -176,9 +176,9 @@ INSERT INTO `message` (`id`, `name`, `secured`, `text_layout_file_name`, `text_t
 
 INSERT INTO `message_i18n` (`id`, `locale`, `title`, `subject`, `text_message`, `html_message`) VALUES
     (@max, 'de_DE', NULL, NULL, NULL, NULL),
-    (@max, 'en_US', 'Mail sent after a subscription to newsletter', 'Your subscription to %store newsletter', NULL, NULL),
-    (@max, 'es_ES', 'Correo enviado después de la suscripción al boletín de noticias', 'Tu suscripción al boletín de %store', NULL, NULL),
-    (@max, 'fr_FR', 'Email envoyé après l\'inscription à la newsletter', 'Vous avez souscrit à la newsletter du site %store', NULL, NULL)
+    (@max, 'en_US', 'Mail sent after a subscription to newsletter', NULL, NULL, NULL),
```
Comment (bibich) [inline setup/update/sql/2.3.0-alpha2.sql]: Same problem here.

File: setup/insert.sql (hunk 1 above)
Comment (roadster31) [inline setup/insert.sql]: What is the problem ?

**Comment by bibich** [comment]
Thanks

**Pull request merged**